ui/components/Navigator.js

```diff
@@ -1091,7 +1098,8 @@ class Navigator extends React.Component {
                   path === href && classes.itemActiveItem
                 )}
                 onClick={() => this.toggleItemCollapse(childId)}
-
+                onMouseOver={() => children && isDrawerCollapsed ? this.setState({ hoveredId : childId }) : null}
+                onMouseLeave={() => !this.state.openItems.includes(childId) ? this.setState({ hoveredId : null }) : null}
```

Comment:
What are we doing with these hoverId states??

Reply:
They are controlling the state of our friendly arrow on hovering.. :)